frontend/components/text-classifier/header/TextClassificationHeader.vue

```diff
@@ -59,6 +62,9 @@ export default {
     isMultiLabel() {
       return this.dataset.isMultiLabel;
     },
+    isRuleListView() {
+      return this.dataset.viewSettings.visibleRulesList;
+    },
```

Comment:
Carefull with null => replace by this.dataset.viewSettings?.visibleRulesList || false